Source/Scene/ImplicitTileCoordinates.js

```diff
@@ -35,12 +37,42 @@ import ImplicitSubdivisionScheme from "./ImplicitSubdivisionScheme.js";
 export default function ImplicitTileCoordinates(options) {
```

Comment:
Hm, maybe the class documentation should include a paragraph explaining what we mean by global (relative to the implicit root tile) and local (relative to the root of a specific subtree) coordinates?